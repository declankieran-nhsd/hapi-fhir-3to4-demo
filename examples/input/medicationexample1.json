{
  "resourceType": "Medication",
  "text": {
    "fhir_comments": [
      "  this example includes the code text element only  "
    ],
    "status": "generated",
    "div": "<div>Amoxicillin 250mg/5ml Suspension</div>"
  },
  "code": {
    "text": "Amoxicillin 250mg/5ml Suspension"
  }
}