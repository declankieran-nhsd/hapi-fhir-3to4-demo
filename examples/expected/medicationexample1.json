{
  "resourceType": "Medication",
  "text": {
    "status": "generated",
    "div": "<div>Amoxicillin 250mg/5ml Suspension</div>"
  },
  "code": {
    "text": "Amoxicillin 250mg/5ml Suspension"
  }
}